tests/compiletest.rs

```diff
@@ -3,7 +3,15 @@ extern crate compiletest_rs as compiletest;
 use std::path::{PathBuf, Path};
 use std::io::Write;
 
+macro_rules! eprintln {
+    ($($arg:tt)*) => {
+        let stderr = std::io::stderr();
+        writeln!(stderr.lock(), $($arg)*).unwrap();
+    }
```

Comment:
You might need an additional pair of {}, but that might have been fixed. Since it works, looks like the latter